External/FEXCore/Source/Interface/Core/Interpreter/InterpreterCore.cpp

```diff
@@ -2371,6 +2386,23 @@ void InterpreterCore::ExecuteCode(FEXCore::Core::InternalThreadState *Thread) {
             memcpy(GDP, Tmp, OpSize);
             break;
           }
+          case IR::OP_VFADDP: {
+            auto Op = IROp->C<IR::IROp_VFAddP>();
+            void *Src1 = GetSrc<void*>(Op->Header.Args[0]);
+            void *Src2 = GetSrc<void*>(Op->Header.Args[1]);
+            uint8_t Tmp[16];
+
+            uint8_t Elements = OpSize / Op->Header.ElementSize / 2;
```

Comment:
Could we have some brackets so I don't have to remember operator precedence rules?

Reply:
Concern addressed